client/src/components/RegisterForm.js

```diff
@@ -0,0 +1,129 @@
+import {
+	FormControl,
+	MenuItem,
+	TextField, Button, Container, Typography } from "@mui/material";
+import styled from "@emotion/styled";
+import React from "react";
+
+//stores the options for the dropdown lists
+const roles = ["Admin", "Student"];
+const regions = ["London", "West Midlands", "Scotland", "Manchester"];
+const classes = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15"];
+
+const RegisterContainer = styled(Container)({
+	background: "#fafafa",
+    width: "max-content",
+    padding: "30px 50px",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+});
+const RegisterButton = styled(Button)({
+	marginTop: "10px",
+});
+
+const RegisterForm = () => {
+    //states for the options in the dropdown lists
+    const [role, setRole] =React.useState("");
+    const [region, setRegion] = React.useState("");
+    const [classNr, setClassNr] = React.useState("");
+
+    //events for updating the value of the selected options
+    const handleChangeRole = (event) => {
```

Comment:
- You may end up with a single event handler function  if you use one **1 useState.** hook.
Which is great!!

Reply:
All this changes have been added in the last PR